votingpool/withdrawal.go

```diff
@@ -707,6 +744,71 @@ func (s *WithdrawalStatus) updateStatusFor(tx *withdrawalTx) {
 	}
 }
 
+// match returns true if the given arguments match the fields in this
+// withdrawalInfo. For the requests slice, the order of the items does not
+// matter.
+func (wi *withdrawalInfo) match(requests []OutputRequest, startAddress WithdrawalAddress,
+	lastSeriesID uint32, changeStart ChangeAddress, dustThreshold btcutil.Amount) bool {
+	// Use reflect.DeepEqual to compare changeStart and startAddress as they're
+	// structs that contain pointers and we want to compare their content and
+	// not their address.
+	if !reflect.DeepEqual(changeStart, wi.changeStart) {
```

Comment:
Fixed. I did run goling, but it seems like it doesn't pick if {} else if {} constructions: https://github.com/golang/lint/blob/master/lint.go#L978